src/app/tests/suites/certification/Test_TC_TSTAT_1_1.yaml

```diff
@@ -178,18 +178,17 @@ tests:
       response:
           constraints:
               type: list
-
-    #issue #15011 disabled steps below Global attributes missing from YAML framework
-    - label: "Read the global attribute: EventList"
-      disabled: true
-      verification: |
-          Not implemented in chip-tool
-      cluster: "LogCommands"
-      command: "UserPrompt"
-      PICS: PICS_USER_PROMPT
-      arguments:
-          values:
-              - name: "message"
-                value: "Please enter 'y' for success"
-              - name: "expectedValue"
-                value: "y"
+    # #issue #15011 disabled steps below Global attributes missing from YAML framework
```

Comment:
@chrisbeach59 we should try to not comment out steps here in the YAMLs. Has this test step been removed from the test plan as well ? If yes , it does make sense to remove it from here otherwise would be impossible to keep track if we directly make changes in the scripts alone. @cjandhyala any comments on how to approach such changes which aren't supported on chip-tool yet ?

Reply:
@raju-apple  The the step is not supported by the chip tool.  I tried to make it disabled, but the presence of the PICS_USER_PROMPT forced it to semi automated.  I am fine with just pulling it out.

Reply:
The `disabled: true` should be making it disabled no matter what... Is that not working?

Reply:
@bzbarsky-apple it does from a SDK CI perspective , but the presence of any step having the disabled tag currently marks this Testcase as a "semi-automated" test in TH.

@chrisbeach59 Whats the specific issue by it being a semi-automated test ? is it the fact that you have an additional prompt to press y/n ? (which in this case is redundant since its anyway not supported by chip-tool?)

Reply:
@raju-apple It does not actually do the prompt.  I am guessing since the step is disabled.  It does flag it as semi-automated I am guessing because the PICS_USER_PROMPT takes precedence over disabled. I was assuming that in the future it will be reading an event list and checking against some list of values.

Reply:
I guess flagging it as semi automated when it runs as automated offended my sensibilities.  lol

Reply:
Once we have events being handled I would believe all yaml scripts that should be checking for eventlist will need to be updated.